src/entities/fractions/percent.rs

```diff
@@ -22,12 +22,14 @@ impl Percent {
     /// Converts the Percent to a string with a specified number of significant digits and rounding strategy
     pub fn to_significant(&self, significant_digits: u8, rounding: Rounding) -> String {
         // Convert the Percent to a simple Fraction, multiply by 100, and then call to_significant on the result
-        (self.as_fraction() * ONE_HUNDRED.clone()).to_significant(significant_digits, rounding)
+        (self.as_fraction() * ONE_HUNDRED.clone())
+            .to_significant(significant_digits, rounding)
+            .unwrap()
     }
 
     /// Converts the Percent to a string with a fixed number of decimal places and rounding strategy
     pub fn to_fixed(&self, decimal_places: u8, rounding: Rounding) -> String {
-        // Convert the Percent to a simple Fraction, multiply by 100, and then call to_fixed on the result
+        // Convert the Percent to a simple Fraction, multiply by 100, and then call to_fixed on the
```

Comment:
```suggestion
        // Convert the Percent to a simple Fraction, multiply by 100, and then call to_fixed on the result
```
Please don't accidentally delete comments.